numpy/core/src/multiarray/compiled_base_pack_inner.h

```diff
@@ -0,0 +1,13 @@
+#ifndef _NPY_PRIVATE__COMPILED_BASE_PACK_INNER_H_
+#define _NPY_PRIVATE__COMPILED_BASE_PACK_INNER_H_
+#include <string.h>
+#include "simd/simd.h"
+#include "lowlevel_strided_loops.h" /* for npy_bswap8 */
```

Comment:
You should only need the headers which define `NPY_VISIBILITY_HIDDEN`, `NPY_NO_EXPORT` and `NPY_CPU_DISPATCH_DECLARE` here, which I think are
```suggestion
#include "npy_cpu_dispatch.h"
#include "npy_common.h"
```

Reply:
`NPY_CPU_DISPATCH_DECLARE` lies in `npy_cpu_dispatch.h`, `NPY_NO_EXPORT` lies in `ndarraytypes.h`